src/main/scala/com/amazon/deequ/profiles/ColumnProfiler.scala

```diff
@@ -33,11 +33,25 @@ private[deequ] case class GenericColumnStatistics(
     knownTypes: Map[String, DataTypeInstances.Value],
     typeDetectionHistograms: Map[String, Map[String, Long]],
     approximateNumDistincts: Map[String, Long],
-    completenesses: Map[String, Double]) {
+    completenesses: Map[String, Double],
+    predefinedColumnDataTypes: Option[Map[String, String]]) {
```

Comment:
Why is a map from String to String and not from String to DataTypeInstances.Value?

Reply:
DataTypeInstances is private to Deequ packages so I could only pass in map from String to String in our spark app. But I guess I could convert it to be from String to DataTypeInstances.Value in extractGenericStatistics method.

Reply:
Might make sense to expose it to the outside (e.g., remove the private scope)

Reply:
I decide to respect this private scope (because I am not sure if there are other reasons why it is set to private) and send a new commit, please feel free to review it.

Reply:
If you think that it could be exposed, then I could update the code accordingly.